subprojects/platform-play/src/integTestPrepare/groovy/org/gradle/play/prepare/PopulateRepositoryPreparationTest.groovy

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.play.prepare
+
+import org.gradle.play.integtest.fixtures.PlayMultiVersionIntegrationTest
+import org.gradle.play.integtest.fixtures.app.BasicPlayApp
+
+class PopulateRepositoryTest extends PlayMultiVersionIntegrationTest {
```

Comment:
I think this doesn't populate everything (e.g., additional jars are needed for `runPlay` and more "advanced" samples).  

I think we should switch this out for the advanced sample and add a custom task that resolves each of the configurations we need.